person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative './nameable'
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..500)
     @name = name
     @age = age
     @parent_permission = parent_permission
```

Comment:
- [ ] Please make sure the person class has a method to add rentals as you created in the book class to work association as expected.